runtime/src/lib.rs

```diff
@@ -106,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 /// up by `pallet_aura` to implement `fn slot_duration()`.
 ///
 /// Change this to adjust the block time.
-pub const MILLISECS_PER_BLOCK: u64 = 12000;
+pub const MILLISECS_PER_BLOCK: u64 = 3000;
```

Comment:
Have you tried this? As standalone chain I arbitrarily set this to 3s as I think we can achieve that target and with some fine tuning and perhaps stricter hardware requirements for our validators perhaps bring it down even more. But in parachain mode I think for the moment we have to stick to the block times of the relay chain. Is my understanding that this will change in the near future so we can go back to faster block production

Reply:
Fixed in https://github.com/valibre-org/vln-node/pull/69/commits/8049eb6647f8f6f16a1fc3ddbb560c251cf9f530